Applications/NodeConf-TLP.md

```diff
@@ -0,0 +1,74 @@
+## Introduction & History
+
+NodeConf was started in 2011 by Mikeal Rogers (@mikeal) and Chris Williams (@voodootikigod).
+
+Today there are several NodeConf's all run by differnet organizers who loosely support each other in the `nodeconf` GitHub org.
+
+## Scope
+
+All current and future NodeConf's:
+
+* NodeConf US (currently in hiatus)
+* NodeConf Adventure
+* NodeConf EU
+* NodeConf London
+* NodeConf Barcelona
+* NodeConf Christchurch
+* NodeConf Santa Cruz
+* NodeConf Buenos Aires
+* NodeConf Austin
+* NodeConf Oakland
+* NodeConf Oslo
+
+Conferences that aren't *called* NodeConf can also be admitted if they choose to be.
+
+## Governance
+
+Individual NodeConf's are considered Working Groups under the Foundation's Governance model.
+
+The NodeConf Community is independently governed by existing conference organizers. The TC (Technical Committee) will use the
+consensus seeking process that has become the standard in Node.js Core for some time.
+
+A single seat on the TC is appointed by each Conference WG.
+
+Because conferences can be significant financial liabilities full autonomy over decision making and finances is granted to
+Working Groups without restrictions on their governance structure. If the TC believes that a Working Group is no longer representing
+the values of the community its only recourse is to revoke the Working Group.
+
+## Contribution Process
+
+Anyone interested in running a NodeConf can create an issue on the `organizers` repo (currently called the `collective` repo).
+If no objections are raised by the TC a new team and repo will be created that is owned by the requesting organizers.
+
+If a conference is not run by its Working Group for more than one year it is considered to be on hiatus and new organizers can ask to
+take on the working group.
+
+## Tools
+
+* GitHub `nodeconf` org.
+* Twitter account `nodeconf`.
+* Facebook and Google+ Pages.
+
+## IP
+
+* NodeConf Trademark, owned by TenConf LLC (@mikeal's LLC)
+
+## TC Members
+
+* Mikeal Rogers (@mikeal)
+* *Awaiting Appointments by other Conference WGs*
+
+## Working Groups
+
+* NodeConf US (currently in hiatus)
+* NodeConf Adventure
+* NodeConf EU
+* NodeConf London
+* NodeConf Barcelona
+* NodeConf Christchurch
+* NodeConf Santa Cruz
+* NodeConf Buenos Aires
```

Comment:
YLD (@dscape) can surely lead London — track record of every single conference in london and many european main events. Happy to donate nodeconf.uk to the foundation too afterwards, like i did with nodeconf.eu.

I think it's appropriate to take this to @nearform too, that should be involved since they do lots of great things for the european market and have an interest in London.